go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/VividCortex/gohistogram v1.0.0 // indirect
 	github.com/btcsuite/btcd v0.0.0-20190523000118-16327141da8c // indirect
 	github.com/btcsuite/btcutil v0.0.0-20190425235716-9e5f4b9a998d
+	github.com/etcd-io/bbolt v1.3.3 // indirect
```

Comment:
Where this dependency came from?

Reply:
I have no idea, it is an indirect dependency, so might be from one of the packages we use. This is a result of [`go mod tidy`](https://github.com/golang/go/issues/27428). Might be required by some of the dependencies for testing purposes :man_shrugging: